### PR TITLE
Allow custom jwksUris for certificates.

### LINKED ISSUE #8: mock-jwks can't be used with all OpenId implementations
createJWKSMock needs a parameter to control whether it should add '/.well-known/jwks.json' or not. Some openid implementations use different url. 
For example for keycloak it is: http://{host}/auth/realms/{realm}/protocol/openid-connect/certs

### PR DESCRIPTION
**Pull request by Levino**
Fix #8: Allow custom jwksUris for certificates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,94 +32,137 @@ const createApp = ({ jwksHost }) => {
...
-      aud: 'private',
-      iss: 'master'
+      iss: 'master',
     })
-    await request.get('/').set('Authorization' , `Bearer ${access_token}`).expect(401)
+    const { status } = await request
+      .get('/')
+      .set('Authorization', `Bearer ${access_token}`)
+    await tearDown({ jwksMock, server })
+    assert.equal(status, 200)
```
Comment (Levino) [inline README.md]: I just copied the code from the examples over.

**Comment by aschekatihin** [comment]
For me it looks excessive. If both parameters are specified first isn't used anywhere, but without looking into sources it is hard to say what they are for, and whether host in the first parameter should be the same as in the second parameter or not.

Technically valid use cases that make no sense:

```
const jwksMock = createJWKSMock(
      '',
      'https://hardfork.eu.auth0.com/protocol/openid-connect/certs'
    )
```

Event worse, which one is used?
```
const jwksMock = createJWKSMock(
      'http://account.google.com/auth/.well-known/jwks.json',
      'https://hardfork.eu.auth0.com/protocol/openid-connect/certs'
    )
```

**Comment by aschekatihin** [comment]
> That is illegal. The first parameter must be a origin, not include a path as it was before. So only

There is no checks. If second parameter provided, it could be literally anything.

Honestly 
```
const jwksMock = createJWKSMock(
      'http://account.google.com',
      'https://hardfork.eu.auth0.com/protocol/openid-connect/certs'
)
```
not any less confusing for me.

```(jwksHost: string, certPath: string = '/.well-known/jwks.json')```
Doesn't brakes API. Anyone using single parameter will continue using it. Anyone who needs two parameters will use two. Test are passing with 1 parameter for this change.

**Comment by Levino** [comment]
How about this?

**Comment by aschekatihin** [comment]
Yes, that's good, thanks!

**Pull request merged**